workers/repomirrorworker/__init__.py

```
@@ -673,3 +705,624 @@ def emit_log(mirror, log_kind, verb, message, tag=None, tags=None, stdout=None,
+    release_org_mirror_config(
+        claimed_config,
+        OrgMirrorStatus.SUCCESS,
+        repos_discovered=total_count,
+        repos_created=newly_created,
+    )
+
+
+def _emit_org_config_log(
+    config: OrgMirrorConfig,
+    log_kind: str,
+    verb: str,
```

Ruff (0.14.13)

Unused function argument: `verb`

(ARG001)